src/d_demoloop.c

```diff
@@ -184,7 +270,20 @@ static void D_GetDefaultDemoLoop(GameMode_t mode)
             break;
     }
 
-    return;
+    if (demoloop && W_CheckNumForName("TITLEPIC") < 0
+        && W_CheckNumForName("DMENUPIC") >= 0)
```

Comment:
This duplicates code with `D_ParsePrimaryLump()`. Any chance to factor this out into a helper function to call after either the DEMOLOOP lump has been parsed or the the default sequence has been selected?

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/2358/commits/37e5cea76a32925938b19a8e92d04873cd43d7c3